assets/js/components/StoreErrorNotice.js

```diff
@@ -25,37 +25,30 @@ import PropTypes from 'prop-types';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
+import ErrorNotice from '../components/ErrorNotice';
 import { STORE_NAME as CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
-import { isPermissionScopeError, isInsufficientPermissionsError } from '../util/errors';
+import { isInsufficientPermissionsError } from '../util/errors';
 import { getInsufficientPermissionsErrorDescription } from '../util/insufficient-permissions-error-description';
-import ErrorText from '../components/error-text';
 const { useSelect } = Data;
 
-function StoreErrorNotice( { moduleSlug, storeName, shouldDisplayError } ) {
+export default function StoreErrorNotice( { storeName, shouldDisplayError, moduleSlug } ) {
```

Comment:
@felixarntz this component is no longer used at all – should we keep it around?

Reply:
@aaemnnosttv I think it can be removed in that case.